app/views/cases/index.html.slim

```diff
@@ -8,7 +8,11 @@
     .column-full.button-holder
       = link_to t('common.new_case_button'), new_case_path, class: 'button'
 
-= render partial: 'state_filter_form' unless request.path == '/cases/my_open'
```

Comment:
We need to add this back in and remove the commented out code for the time being